eox_tenant/templatetags/ednx.py

```diff
@@ -225,4 +225,8 @@ def tenant_get_value(value, *args, **kwargs):  # pylint: disable=unused-argument
     Django template filter that wraps the configuration_helpers.get_value function
     """
     default = kwargs.get('default', None)
-    return configuration_helpers.get_value(value, default)
+
+    if getattr(settings, 'USE_EOX_TENANT', False):
```

Comment:
@magajh would you rather use this if? or something more like https://github.com/eduNEXT/eox-tenant/blob/5f42d52d6083c66e306cfa6fb5606d367ee52500/eox_tenant/templatetags/ednx.py#L164?

we could go for
```
configuration_helpers.get_value(value, getattr(settings, value, default))
```

Reply:
Also, can you please change `microsite_get_value` so that it internally calls this method?

Reply:
I changed it to `configuration_helpers.get_value(value, getattr(settings, value, default))`